cabal-install/Distribution/Client/ProjectPlanning.hs

```diff
@@ -1,4 +1,4 @@
-{-# LANGUAGE CPP, RecordWildCards, NamedFieldPuns, RankNTypes #-}
+{-# LANGUAGE RecordWildCards, NamedFieldPuns, RankNTypes #-}
 {-# LANGUAGE ViewPatterns #-}
```

Comment:
@23Skidoo that's btw a very common misconception... it's too late now, but I wonder if `depend-extensions` would have been a better name.